src/code/z_play.c

```diff
@@ -1463,7 +1464,9 @@ void Play_SpawnScene(PlayState* this, s32 sceneId, s32 spawn) {
 
     Play_InitScene(this, spawn);
 
-    osSyncPrintf("ROOM SIZE=%fK\n", func_80096FE8(this, &this->roomCtx) / 1024.0f);
+    size = func_80096FE8(this, &this->roomCtx);
+
+    osSyncPrintf("ROOM SIZE=%fK\n", size / 1024.0f);
```

Comment:
whats the motivation here? think it looked fine as is but not sure if theres a reason im missing

Reply:
That func_ call is doing something important about room loading (I forgot the details) and it's better not hidden in a printf, for readability

Also it's good practice not putting code with side effects in (debug) printfs, again for readability and possibly if one was to macro out printf calls in a non-debug build
(I don't know if that's the case for retail)

Reply:
fair enough. Yeah it does do the actual allocation of the room buffers, not just get their size